.github/workflows/terratest.yml

```diff
@@ -28,6 +28,11 @@ jobs:
         with:
           go-version: 1.18
         id: go
+      - name: Set up Terraform
+        uses: hashicorp/setup-terraform@v2
+        with:
+          cli_config_credentials_token: ${{ secrets.TF_API_TOKEN }}
+          terraform_version: 1.3.0
```

Comment:
Is this required?  Doesn't default to latest?

Reply:
lol, I cannot fix the terratest, so deleted PR